x-pack/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -62,7 +62,7 @@ export function getPathAsArray(pathname: string = '') {
 }
 
 export function removeUndefinedProps<T>(obj: T): Partial<T> {
```

Comment:
You can avoid `any` by updating this line:
```suggestion
export function removeUndefinedProps<T extends object>(obj: T): Partial<T> {
```

Reply:
👍